CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+## Welcome
+
+LocalStack is a community driven project that aims to help developers emulate AWS services locally, without connecting to any cloud provider! The community version of LocalStack has grown exceptionally and now supports various AWS services, which is detailed on our [feature coverage](https://docs.localstack.cloud/aws/feature-coverage/). 
+
+This document serves the need for a guide for contributors to help them understand the overall organization of the LocalStack community projects. As an emerging open-source community, we have a plethora of work that we need to do and complete, and it is not just limited to code. 
+
+From a contribution perspective, you can work on writing examples, triaging issues and pull requests, submitting bug fixes, suggesting improvements and making our documentation accessible to more users. 
+
+## Prerequisites
+
+Before submitting any contributions to LocalStack, you should complete the following prerequisties. Having completed and understood the basic philosophy around these steps would make it easier for you to land your first contribution.
+
+### Code of conduct
+
+LocalStack governs its participants according to the Contributor Covenant [Code of Conduct](CODE_OF_CONDUCT.md). As a contributor, you agree to uphold this code. Please report unacceptable behavior to [info@localstack.cloud](mailto:info@localstack.cloud).
+
+### Setup the development environment
+
+To contribute code contributions, setting up a development environment is required. Review the [LocalStack developer guide](https://docs.localstack.cloud/developer-guide/) to understand how you can setup the environment and make the desirable changes. It is also recommended to look at our [public roadmap](https://github.com/localstack/localstack/blob/master/doc/roadmap/README.md) and [open issues](https://github.com/localstack/localstack/issues) for better context around contributions we are looking for.
```

Comment:
Not sure if/how we're using the road map. ` Jul 19, 2017` => seems like this document is quite abandoned at the moment.

Reply:
I would remove its references from the LocalStack `README.md` as well then.